dts/bindings/serial/renesas,smartbond-uart.yaml

```diff
@@ -0,0 +1,38 @@
+description: Renesas SmartBond(tm) UART
+
+compatible: "renesas,smartbond-uart"
+
+include: [uart-controller.yaml, pinctrl-device.yaml]
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: true
+
+    peripheral-id:
+      type: int
+      description: |
+        UART peripheral identifier. 0 for UART, 1 for UART2 and 2 for UART3.
+      required: true
+
```

Comment:
I don't think this is necessary since there are 3 physical instances of UART block, but there's only single GPIO block with pins grouped into 2 ports - p0 and p1 - so there are 2 nodes in device tree for each port